### PR TITLE
Add anchor links to Little Book of Privacy

### PR DESCRIPTION
**Pull request by maxxcrawford**
## Description
Add anchor links to all `h2` and `h3` elements on the Little Book of Privacy page. 

~Blocking merge until Liz approves~ Liz approved! 

## Issue / Bugzilla link

#10079

## Testing

The following links should jump down the page to their respective sections: 

- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#risks
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#data-breaches
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#web-never-forgets
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#trackers-everywhere
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#trolls
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#fake-news
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#control
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#tips
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#starting-over
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#tidy-browser
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#convenience
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#sharing
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#the-world
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#stay-safe
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#for-everyone
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#safer-connections
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#bluetooth
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#https
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#email
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#click-bait
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#bigger-picture
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#location
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#delete
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#trackers
- https://www-demo-pr-10138.herokuapp.com/firefox/privacy/book/#feedback

### CODE REVIEW

**Comment by maxxcrawford** [comment]
_Note for the reviewer: I took editorial license on some of these. Feel free to comment on these._

File: bedrock/firefox/templates/firefox/privacy/book.html [hunk 1]
```diff
@@ -87,11 +87,11 @@ <h3>{{ ftl('privacy-book-the-web-never') }}</h3>
     aspect_ratio='mzp-has-aspect-1-1',
     class='mzp-l-card-feature-right-half mzp-t-firefox'
   ) %}
-    <h3>{{ ftl('privacy-book-trackers-everywhere-') }}</h3>
+    <h3 id="trackers-everywhere">{{ ftl('privacy-book-trackers-everywhere-') }}</h3>
```
Comment (maxxcrawford) [inline bedrock/firefox/templates/firefox/privacy/book.html:90]: There is another link at the bottom of this page that is just `trackers`. Not sure if that's confusing.

File: bedrock/firefox/templates/firefox/privacy/book.html [hunk 2]
```diff
@@ -248,26 +248,26 @@ <h3>{{ ftl('privacy-book-safer-connections-public') }}</h3>
...
   <div class="mzp-l-content mzp-t-content-md">
-    <h3>{{ ftl('privacy-book-email-accounts-and') }}</h3>
+    <h3 id="email">{{ ftl('privacy-book-email-accounts-and') }}</h3>
     <p>{{ ftl('privacy-book-your-email-address') }}</p>
     <img src="{{ static('img/firefox/privacy/book/privacy-identities.png') }}" alt="">
   </div>
 
   <div class="mzp-l-content mzp-t-content-md">
-    <h3>{{ ftl('privacy-book-resist-the-urge') }}</h3>
+    <h3 id="click-bait">{{ ftl('privacy-book-resist-the-urge') }}</h3>
```
Comment (maxxcrawford) [inline bedrock/firefox/templates/firefox/privacy/book.html:270]: I used the term `click-bait` here.

**Review by maxxcrawford — COMMENTED** [review]
_Note for the reviewer: I took editorial license on some of these. Feel free to comment on these._

**Review by craigcook — APPROVED** [review]
🔖

**Pull request merged**